### PR TITLE
Fix Reporting in email

### PR DESCRIPTION
**Pull request by vavuthu**
Fetch container name instead of parging image for getting name

Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

File: ocs_ci/framework/pytest_customization/ocscilib.py [hunk 1]
```diff
@@ -137,8 +137,8 @@ def pytest_configure(config):
             # add csi versions
             csi_versions = get_csi_versions()
             config._metadata['csi-provisioner'] = csi_versions.get('csi-provisioner')
```
Comment (petr-balogh) [inline ocs_ci/framework/pytest_customization/ocscilib.py]: csi-provisioner was also None I saw, but do not see this fix is touching this one?

**Comment by vavuthu** [comment]
Verified with Downstream build:

```
cephfsplugin | 4.2-93.279197bc.master
csi-provisioner | 4.2
rbdplugin | 4.2-93.279197bc.master

```

verified with upstream build:

```
cephfsplugin | v1.2.0
csi-provisioner | v1.3.0
rbdplugin | v1.2.0

```

File: ocs_ci/framework/pytest_customization/ocscilib.py (hunk 1 above)
Comment (vavuthu) [reply ocs_ci/framework/pytest_customization/ocscilib.py]: I didn't see any reason why csi-provisioner is None in email where this problem is initiated. I have tested with bot upstream and downstream builds with this PR and its able to get the version as expected.

**Comment by petr-balogh** [comment]
Raz already Approved but after Vijay rebase he lost his approve. So merging

**Pull request merged**